testing/sysprof/APKBUILD

```diff
@@ -0,0 +1,38 @@
+# Contributor: Rasmus Thomsen <oss@cogitri.dev>
+# Maintainer: Rasmus Thomsen <oss@cogitri.dev>
+pkgname=sysprof
+pkgver=3.32.0
+pkgrel=0
+pkgdesc="System-wide profiler for Linux"
+url="http://sysprof.com/"
+arch="all"
+license="GPL-2.0-or-later"
+depends_dev="$pkgname=$pkgver-r$pkgrel" # sysprof's lib setup is a bit weird...
+makedepends="meson gtk+3.0-dev itstool glib-dev polkit-dev elogind-dev"
+subpackages="$pkgname-dev $pkgname-doc"
```

Comment:
abuild says we probably need to add `$pkgname-static` and `$pkgname-lang` here.

Reply:
Ah, thanks, didn't see that warning. Will add